clients/src/main/java/org/apache/kafka/common/requests/ListTransactionsRequest.java

```diff
@@ -38,6 +38,10 @@ public ListTransactionsRequest build(short version) {
                 throw new UnsupportedVersionException("Duration filter can be set only when using API version 1 or higher." +
                         " If client is connected to an older broker, do not specify duration filter or set duration filter to -1.");
             }
+            if (data.transactionalIdPattern() != null && version < 2) {
+                throw new UnsupportedVersionException("Transactional ID pattern filter can be set only when using API version 1 or higher." +
+                    " If client is connected to an older broker, do not specify the pattern filter.");
```

Comment:
Not sure if I misunderstood anything, but Transactional ID pattern is introduced in api version 2+ right?
Doe that mean the error message should be  `...using API version 2 or higher.` ?